tasks.py

```diff
@@ -1115,6 +1133,8 @@ def service(
                 "up",
                 "-d",
                 "--quiet-pull",
+                "--wait",
+                "--wait-timeout 90",
```

Comment:
Waits for successful heathchecks or finished containers before proceeding to the next step. Times out and raises an error at 90 seconds for unhealthy containers.